graph/src/data/query/error.rs

```diff
@@ -78,10 +93,59 @@ impl fmt::Display for QueryExecutionError {
                 f,
                 "Only a single top-level field is allowed in subscriptions"
             ),
+            QueryExecutionError::SupgraphIdError(s) => {
+                write!(f, "Failed to get subgraph ID from type: {}", s)
+            }
+            QueryExecutionError::RangeArgumentsError(s) => write!(
+                f,
+                "Range inputs must be an integer, {:}",
```

Comment:
Three improvements here:

* Use a `:` instead of a `,` to separate the higher-level message from the list of arguments.
* Call them `arguments`, not `inputs`.
* Use `graph_graphql::schema::validation::Strings` to obtain a CSV-formatted list of strings instead of using `fold`: https://github.com/graphprotocol/graph-node/blob/4e0478648230c32735d4c02ca8208a9421b67f25/graphql/src/schema/validation.rs#L7-L15 (we should move this into `graph::util` but for now pulling it from `validation` is ok).

Reply:
You could also just do `strings.join(", ")` here if you want. It's probably not worth pulling in `Strings`.